src/sortStudents.ts

```diff
@@ -1,16 +1,51 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+function getAverageValue(numbers: number[]): number {
+  return numbers.reduce((prev, curr) => prev + curr) / numbers.length;
```

Comment:
```suggestion
  return numbers.reduce((prev, curr) => prev + curr, 0) / numbers.length;
```
Add starting value every time, it's safer and bug proof.